tests/configfiles/DockerfileLintBearTest.py

```diff
@@ -24,6 +26,71 @@
 """
 
 
+good_file_2 = """
+FROM ubuntu
+MAINTAINER coala
+LABEL Name coala-docker
+LABEL Version 0.1
+
+RUN apt-get -y -qq update
+RUN apt-get -y -qq upgrade
+RUN curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+EXPOSE 5432
+CMD ["/usr/bin/node", "/var/www/app.js"]
+"""
+
+
+bad_file_2 = """
+FROM ubuntu
+
+apt-get -y -qq update
+apt-get -y -qq upgrade
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+"""
+
+good_file_3 = """
+FROM kali
+MAINTAINER coala
+LABEL Name coala-docker
+LABEL Version 0.1
+EXPOSE 5432
+"""
+
+
+bad_file_3 = """
+FROM kali
+LABEL Name
+LABEL Version
+"""
+
+default_rule = os.path.join(os.path.dirname(__file__),
+                            'dockerfile_lint_test_files',
+                            'default_rule.yaml')
+
+sample_rule = os.path.join(os.path.dirname(__file__),
+                           'dockerfile_lint_test_files',
+                           'sample_rule.yaml')
```

Comment:
You could write a function `get_testfile_name` that does the path-join (many tests do the same). But optional.